packages/security/gnutls/package.mk

```diff
@@ -25,3 +26,8 @@ PKG_CONFIGURE_OPTS_TARGET="--disable-doc \
                            --with-included-unistring \
                            --without-p11-kit \
                            --without-tpm"
+
+post_configure_target() {
+  sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+  sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+}
```

Comment:
I think it would look nicer to use `sed -i -e '....' -e '....' file` when actions are set on same file.

Reply:
Whilst neater, I went with the 2 lines as it mirrors the guide - https://docs.fedoraproject.org/en-US/packaging-guidelines/#_removing_rpath - ideally we can get upstream fixed, and use the `—disable-rpath` with `configure` and subsequently with `libtool`. Have started the process of upstream fixes with `gettext`